### PR TITLE
[21474] Long words are not punctuated in WP split view

### PR DESCRIPTION
**Pull request by HDinger**
This sets the overflow-values so that long words are punctuated.

https://community.openproject.org/work_packages/21474

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hdinger-fix-21474-long-words-not-punctuated-wp-split-view.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://hdinger-fix-21474-long-words-not-punctuated-wp-split-view.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by HDinger** [comment]
Before:

long_text

After:

long_text_correct

**Comment by NobodysNightmare** [comment]
The approach you've chosen will not allow me to read those long words I am using:

image

 This is bad because
1. long words are legitimate
2. the comment view is rather slim

Ideally the browser would correctly hyphenate those words, which can be indicated by `hyphens: auto`. However, this is not very well supported and also requires correct `lang` annotation of the content. So it is probably no option for us.

The not-so-ideal-but-somewhat-readable alternative is using `overflow-wrap: break-word`:

image

This at least allows to read arbitrarly long words, even though they are not correctly hyphenated...

**Comment by NobodysNightmare** [comment]
For custom fields you optimized for the edge case too much, therefore breaking the normal case.

Before this PR:

image

After this PR:

image

Again I propose to use `overflow-wrap`:

image

**Comment by NobodysNightmare** [comment]
Note: In IE you will need to use `word-wrap` instead of `overflow-wrap`. Not sure if this conversion is done by the autoprefixer... (because it is called "prefixer")

File: app/assets/stylesheets/content/work_package_details/_activities_tab.sass [hunk 1]
```diff
@@ -42,6 +42,17 @@
...
+  li
+    span
+      word-wrap: break-word
+      // important for IE
+      display: inline-block
+      max-width: 100%
+      vertical-align: top
+  p
+    overflow: hidden
+    text-overflow: ellipsis
```
Comment (NobodysNightmare) [inline app/assets/stylesheets/content/work_package_details/_activities_tab.sass]: what about word wrapping for `p` elements?

**Comment by NobodysNightmare** [comment]
Looking good, will wait for travis. :horse:

**Pull request merged**